src/login/Login.jsx

```diff
@@ -8,7 +8,7 @@ import { SERVICE_UNAVAILABLE_ERROR } from "../messages";
 export default function Login() {
     const [errorMessage, setErrorMessage] = useState(null);
     const {isAuth,login} = useContext(AuthContext);
-    const [user, setUser] = useState(null);
+    const [, setUser] = useState(null);
```

Comment:
don't remove this, you can use this to store the username also.

Reply:
@Aryamanz29, can you please submit the new change as requested by @codesankalp here?

Reply:
@mtreacy002 Done 😄